src/main/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtils.java

```diff
@@ -0,0 +1,492 @@
+     * null if all common contigs are equivalent
+     *
+     * @param commonContigs
+     * @param dict1
+     * @param dict2
+     * @return
+     */
+    private static List<SAMSequenceRecord> findDisequalCommonContigs(Set<String> commonContigs, SAMSequenceDictionary dict1, SAMSequenceDictionary dict2) {
+        for ( String name : commonContigs ) {
+            SAMSequenceRecord elt1 = dict1.getSequence(name);
+            SAMSequenceRecord elt2 = dict2.getSequence(name);
+            if ( ! sequenceRecordsAreEquivalent(elt1, elt2) )
+                return Arrays.asList(elt1,elt2);
+        }
+
+        return null;
+    }
+
+    /**
+     * Helper routine that returns two sequence records are equivalent, defined as having the same name and
+     * lengths, if both are non-zero
+     *
+     * @param me
+     * @param that
+     * @return
+     */
+    private static boolean sequenceRecordsAreEquivalent(final SAMSequenceRecord me, final SAMSequenceRecord that) {
+        if (me == that) return true;
+        if (that == null) return false;
+
+        if (me.getSequenceLength() != 0 && that.getSequenceLength() != 0 && me.getSequenceLength() != that.getSequenceLength())
+            return false;
+
+        // todo -- reenable if we want to be really strict here
+//        if (me.getExtendedAttribute(SAMSequenceRecord.MD5_TAG) != null && that.getExtendedAttribute(SAMSequenceRecord.MD5_TAG) != null) {
+//            final BigInteger thisMd5 = new BigInteger((String)me.getExtendedAttribute(SAMSequenceRecord.MD5_TAG), 16);
+//            final BigInteger thatMd5 = new BigInteger((String)that.getExtendedAttribute(SAMSequenceRecord.MD5_TAG), 16);
+//            if (!thisMd5.equals(thatMd5)) {
+//                return false;
+//            }
+//        }
+//        else {
+        if (me.getSequenceName() != that.getSequenceName())
+            return false; // Compare using == since we intern() the Strings
+//        }
+
+        return true;
+    }
+
+    /**
+     * A very simple (and naive) algorithm to determine (1) if the dict is a human reference (hg18/hg19) and if it's
+     * lexicographically sorted.  Works by matching lengths of the static chr1, chr10, and chr2, and then if these
+     * are all matched, requiring that the order be chr1, chr2, chr10.
+     *
+     * @param dict
+     * @return
+     */
+    private static boolean nonCanonicalHumanContigOrder(SAMSequenceDictionary dict) {
+        if ( ! ENABLE_LEXICOGRAPHIC_REQUIREMENT_FOR_HUMAN ) // if we don't want to enable this test, just return false
+            return false;
+
+        boolean human = false;
+        // determine whether the dict is a human reference
+        for ( SAMSequenceRecord elt : dict.getSequences() ) {
+            if (elt.getSequenceName().equals("1") || elt.getSequenceName().equals("chr1")) {
+                human = true;
+                break;
+            }
+        }
```

Comment:
This check is misleading -- having a contig named "1" or "chr1" does not actually mean that the dictionary is a human reference. Recommend removing this loop and `human` boolean entirely, and instead modifying `isHumanSeqRecord()` below to check for a match on the name of the contig in addition to the length.